### PR TITLE
fix(ru): remove duplicate headings

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove duplicate headings. Since I don't speak Russian, please feel free to correct my mistakes.

### Motivation

Prepare for enabling [the `no-duplicate-heading` markdown-lint rule](https://github.com/mdn/translated-content/blob/d13c47b2b819b24e7623bb55ad5f24606c537d87/.markdownlint.jsonc#L20-L22).
Part of #17582

### CODE REVIEW

File: files/ru/web/api/eventsource/index.md [hunk 1]
```diff
@@ -11,11 +11,6 @@ slug: Web/API/EventSource
 
 См. [Using server-sent events](/en/Server-sent_events/Using_server-sent_events) для более детальной информации
 
-## Методы
-
-| `void close();` |
-| --------------- |
-
```
Comment (yin1999) [inline files/ru/web/api/eventsource/index.md:18]: removed, as we never use IDL definition in documents (check the "methods" section below)

File: files/ru/web/css/visual_formatting_model/index.md [hunk 2]
```diff
@@ -43,9 +43,9 @@ _Модель визуального форматирования_ CSS - это
 
 Блоки-контейнеры содержат либо только инлайн-боксы, либо только элементы блочного уровня. Но, как правило, документ содержит и те и другие. В этом случае анонимные блок-боксы создаются вокруг примыкающих к ним инлайн-боксов.
 
-### Пример
+### Пример 1
```
Comment (yin1999) [inline files/ru/web/css/visual_formatting_model/index.md:46]: Temporarily use different numbers to distinguish headings

File: files/ru/web/html/element/dfn/index.md [hunk 3]
```diff
@@ -43,15 +43,6 @@ slug: Web/HTML/Element/dfn
...
 
-## Примечания по использованию
-
-- Элемент `<dfn>` отмечает определяемый термин; определение термина должно быть дано окружающим {{HTMLElement("p")}}, {{HTMLElement("section")}} или группой списков определений (обычно пара {{HTMLElement("dt")}}{{HTMLElement("dd")}}).
-- Точное значение определяемого термина определяется следующими правилами:
-
-  1. Если элемент `<dfn>` имеет [`title`](/ru/docs/Web/HTML/Global_attributes#title) атрибут, то термин является значением этого атрибута.
-  2. Иначе, если он содержит только {{HTMLElement("abbr")}} элемент с [`title`](/ru/docs/Web/HTML/Global_attributes#title) атрибутом, тогда термин является значением этого атрибута. Это продемонстрировано в [Использование сокращений и определений вместе](#использование_сокращений_и_определений_вместе) ниже.
-  3. В противном случае текстовое содержимое `<dfn>` элемента является определяемым термином.
-
```
Comment (yin1999) [inline files/ru/web/html/element/dfn/index.md:54]: remove the duplicate part

File: files/ru/web/javascript/reference/statements/import/index.md [hunk 4]
```diff
@@ -138,40 +132,40 @@ import myDefault, { foo, bar } from "/modules/my-module.js";
 
 Такой код выведет ошибку:
 
-##### my-module.js
+- my-module.js:
```
Comment (yin1999) [inline files/ru/web/javascript/reference/statements/import/index.md:135]: this part has been removed from en-US, but for now, I've used unordered list instead

**Review by leon-win — APPROVED** [review]
Thank you, @yin1999 ! :+1:

**Pull request merged**